include/aos/common/crypto.hpp

```diff
@@ -595,6 +606,15 @@ class ProviderItf {
      */
     virtual Error ASN1DecodeOID(const Array<uint8_t>& inOID, Array<uint8_t>& dst) = 0;
 
+    /**
+     * Creates SHA1 UUID(version 5) based on a given name space and data.
+     *
+     * @param namespace name space.
```

Comment:
there's no 'namespace' param. Please make it consistent.

Reply:
done